blog/02-01-2023-creating-smart-contracts-for-liquidity-farming/creating-smart-contracts-for-liquidity-farming.md

```diff
@@ -0,0 +1,222 @@
+---
+title: Creating Smart Contracts for Liquidity Farming  
+description: Describing Smart Contract For Liquidity Farming and the technical side of creating them
+authors:
+  - name: Adewale Emmanuel
+    title: Software Developer, Technical writer @Celo Foundation
+    url: https://github.com/Manuel-dre
+    image_url: https://avatars.githubusercontent.com/u/69092079?s=400&u=f34c84ee03afb9a51b163652b750419e98ed7456&v=4
+tags: [celo, intermediate, celosage, solidity, 'smartcontract']
+hide_table_of_contents: true
+slug: /tutorials/creating-smart-contracts-for-liquidity-farming
+---
+
+## CREATING SMART CONTRACTS FOR LIQUIDITY FARMING
+
+![header](images/celo-smart-contracts.jpg)
```

Comment:
`![header](../../src/data-tutorials/showcase/intermediate/creating-smart-contracts-for-liquidity-farming.png)`